state-synchronizer/src/tests/fuzzing.rs

```diff
@@ -81,7 +81,8 @@ pub fn test_state_sync_msg_fuzzer_impl(msg: StateSynchronizerMsg) {
         config.upstream,
         MockExecutorProxy::new(SynchronizerEnvHelper::default_handler(), storage_proxy),
         initial_state,
-    );
+    )
+    .unwrap();
```

Comment:
I think the linter will yell at us because this is not an expect :o although I don't see it yelling at us, humm...

Reply:
Yeah.. it doesn't seem to be saying anything? Anyway, I've updated to an expect.